src/lib/stateMappings/__tests__/singleIndex-test.ts

```diff
@@ -0,0 +1,133 @@
+import singleIndexStateMapping from '../singleIndex';
+
+describe('singleIndexStateMapping', () => {
+  describe('stateToRoute', () => {
+    it('passes normal state through', () => {
+      const stateMapping = singleIndexStateMapping('indexName');
+      const actual = stateMapping.stateToRoute({
+        indexName: {
+          query: 'zamboni',
+          refinementList: {
+            color: ['red'],
+          },
+        },
```

Comment:
We can add other indices in there to make sure that it only picks up the provided index properties.